tools/src/main/scala/scala/scalanative/build/NativeConfig.scala

```diff
@@ -117,6 +123,12 @@ sealed trait NativeConfig {
   /** Create a new config with the given lto mode. */
   def withLTO(value: LTO): NativeConfig
 
+  /** Create a new config with the given max inline depth. */
+  def withMaxInlineDepth(value: Int): NativeConfig
+
+  /** Create a new config with the max caller size. */
+  def withMaxCallerSize(value: Int): NativeConfig
```

Comment:
Can we put them in a dedicated object, eg. `OptimizerConfig` 
I think we can then have 
```
trait NativeConfig {
...
def optimizerConfig: OptimizerConfig
def withOptimizerConfig(mapping: OptimizerConfig => OptimizerConfig): NativeConfig
...
```

Reply:
This looks like it got put on the `Config` not the `NativeConfig`.